src/Redux/Features/Task/taskService.ts

```diff
@@ -12,6 +12,7 @@ type Task = {
 }
 
 type TaskResponse = {
+  map(arg0: (activity: any, index: number) => import("react/jsx-runtime").JSX.Element): import("react").ReactNode;
```

Comment:
use a proper types, avoid the use of "any" you know the structure the response already